CMakePresets.json

```diff
@@ -0,0 +1,592 @@
+{
+  "version": 3,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 20,
+    "patch": 0
```

Comment:
Bad news:

File version must be 3 or higher for condition support like this:

```json
      "condition": {
        "type": "equals",
        "lhs": "${hostSystemName}",
        "rhs": "Linux"
      }
```

Version 3 was introduced in CMake 3.21.1. 

Also `installDir` is allowed in preset files specifying version 3 or above.

Reply:
Ah ok, I was just curious why that particular version. It doesn't make much difference since it's not imposing the restriction on the main CMakeLists.txt.